docs/timeseries/index.rst

```diff
@@ -69,7 +69,6 @@ Time series are specialized kinds of |Table| objects::
     2009-05-02T00:48:32.291  6.632337e-04 ...  1.5272264e-03 -1.4998110e-03
     2009-05-02T00:49:31.149  6.632584e-04 ...  1.5193661e-03 -1.5074468e-03
                         ...           ... ...            ...            ...
-    2009-05-11T17:58:22.526  1.014493e-03 ...  3.6121816e-03  3.1950327e-03
```

Comment:
Yes, the usual annoyance that, sadly, doctests just stop when an error is encountered, so you don't necessarily get all of them.

Reply:
ah that's why I needed two iterations ! thanks for clarifying this !